spec/services/sign_in/openid_connect_certificates_presenter_spec.rb

```diff
@@ -6,12 +6,31 @@
   describe '#perform' do
     subject { SignIn::OpenidConnectCertificatesPresenter.new.perform }
 
-    let(:public_key_jwks) { { keys: [public_key_jwk] } }
-    let(:public_key_jwk) {  JWT::JWK.new(public_key, { alg: 'RS256', use: 'sig' }).export }
+    let(:public_key_jwk) { JWT::JWK.new(public_key, { alg: 'RS256', use: 'sig' }).export }
     let(:public_key) { OpenSSL::PKey::RSA.new(File.read(Settings.sign_in.jwt_encode_key)).public_key }
 
-    it 'returns JWK representation of sign in service access token encode key' do
-      expect(subject).to eq(public_key_jwks)
+    shared_examples 'certificates return' do
+      before { Settings.sign_in.jwt_old_encode_key = old_encode_key_setting }
```

Comment:
You should stub instead of updating singletons (prob not an issue here but this is technically test pollution if someone else were to use `jwt_old_encode_key`). Do this instead:
`allow(Settings.sign_in).to receive(:jwt_old_encode_key).and_return(old_encode_key_setting)`